games/minimal/mods/testformspec/mod.conf

```diff
@@ -0,0 +1,2 @@
+name = testformspec
```

Comment:
~~test~~ - probably put this in a modpack named test, but having test in all names is irritating and redundant...